optimum/intel/utils/import_utils.py

```diff
@@ -33,6 +33,7 @@
 STR_OPERATION_TO_FUNC = {">": op.gt, ">=": op.ge, "==": op.eq, "!=": op.ne, "<=": op.le, "<": op.lt}
 
 _optimum_version = importlib_metadata.version("optimum")
+_optimum_intel_version = importlib_metadata.version("optimum-intel")

```

Comment:
why not from optimum.intel.version import __version__?

Reply:
I liked the consistency with _optimum_version :) But also the importlib_metadata version gives the commit ID:

```
>>> from optimum.intel.version import __version__
>>> from optimum.intel.utils.import_utils import _optimum_intel_version
>>> __version__
'1.17.0.dev0'
>>> _optimum_intel_version
'1.17.0.dev0+9d58b66'
```